src/jsx-parser.ts

```diff
@@ -256,9 +266,12 @@ export class JSXParser extends Parser {
         }
 
         this.scanner.throwUnexpectedToken();
+
+        /* istanbul ignore next */
+        throw new Error('Unreachable');
```

Comment:
Without the line, the TypeScript compiler will complain: _error TS2366: Function lacks ending return statement and return type does not include 'undefined'_.

@DanielRosenwasser Could this be another problem with the flow analysis? `throwUnexpectedToken()` does nothing but to throw an exception, thus the compiler should not treat this function as terminated with an undefined return value.